server/liveblog/blogs/schema.py

```diff
@@ -41,6 +41,9 @@
     'public_url': {
         'type': 'string'
     },
+    'public_urls': {
```

Comment:
the `s3` urls needs to be stored at some level.
I choose to stored them in `blog` level instead of the `output` level, because it is also near to the `multiple themes` implementation.